commit 584668415039adeed073decee7e04de28248afd3
Author: fanquake <fanquake@gmail.com>
Date:   Tue Aug 18 01:20:24 2020 +0000

    Disable threading to fix non-determinism

    A bug in the file parser can cause dependencies to be calculated
    differently based on which files have already been parsed. This is more
    likely to occur on systems with more CPUs.

    Just disable threading for now. There is no noticeable slowdown.

    See #9891.

diff --git a/cctools/ld64/src/ld/InputFiles.h b/cctools/ld64/src/ld/InputFiles.h
index ef9c756..90a70b6 100644
--- a/cctools/ld64/src/ld/InputFiles.h
+++ b/cctools/ld64/src/ld/InputFiles.h
@@ -25,7 +25,6 @@
 #ifndef __INPUT_FILES_H__
 #define __INPUT_FILES_H__

-#define HAVE_PTHREADS 1

 #include <stdlib.h>
 #include <sys/types.h>
